examples/patterns/logo-strip.html

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
This file should be `examples/templates/logo-strip.html` as it's a use-case, not a pattern example.